src/query/service/src/databases/share/share_database.rs

```diff
@@ -249,13 +247,6 @@ impl Database for ShareDatabase {
         ))
     }
 
```

Comment:
share database should impl any `writeable` function, and return `PermissionDenied`, so as `retryable_update_multi_table_meta`